src/com/tambolaassingment/Player.java

```diff
@@ -2,16 +2,18 @@
 
 import java.util.HashMap;
 import java.util.Iterator;
+import java.util.Map;
 
 class Player implements Runnable {
-    private TicketChecker ticketChecker;
-    public static  boolean housieStatus = false;
+    private int hitCounter;
+    public static boolean housieStatus = false;
```

Comment:
Maintain Order Class Members by Scopes
You can refer [https://www.oracle.com/technetwork/java/codeconventions-150003.pdf](url)

Reply:
done.